ethereum-consortium/docs/setupWalkthrough.md

```diff
@@ -87,11 +87,11 @@
       resource and copy the IP.  
 
       Navigate to http://{copied ip}:3001/enodes. Save the returned value
-      {bootnodes}.
```

Comment:
Not sure static nodes is the way to go unless that's the registry and the registry is HA. https://github.com/ethereum/go-ethereum/wiki/Connecting-to-the-network#static-nodes

However if this is purely for Dev env than OK. Perhaps some clarification as Static Nodes are expected to ALWAYS be available